docs/conversion_examples_gallery/conversion_example_gallery.rst

```diff
@@ -93,6 +93,7 @@ Behavior
     DeepLabCut <behavior/deeplabcut>
     FicTrac <behavior/fictrac>
     LightningPose <behavior/lightningpose>
+    Neuralynx NVT <behavior/neuralynx_nvt>
```

Comment:
Should we move it further up closer to the other Neuralynx one? It was kind of hard to spot otherwise

Reply:
It's currently in alphabetical order. We could add a note in the other Neuralynx page that points to this one

Reply:
Oh, hmm... I see now on the side

I was referring to the main nav tree on the left bar

![image](https://github.com/catalystneuro/neuroconv/assets/51133164/355337fc-4632-4117-b4d3-554fb464c782)

but I guess that's just the flattened version?

Should definitely have a pointer since someone might have clicked the first Neuralynx one on the left

Reply:
yeah, I think we should get rid of that side bar, but that's a separate issue